sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -534,8 +541,22 @@ def _from_generated(cls, generated):
         blob.blob_tier_inferred = generated.properties.access_tier_inferred
         blob.archive_status = generated.properties.archive_status
         blob.blob_tier_change_time = generated.properties.access_tier_change_time
+        blob.tag_count = generated.properties.tag_count
+        blob.blob_tags = blob._parse_tags(generated.blob_tags)  # pylint: disable=protected-access
         return blob
 
+    @classmethod
+    def _parse_tags(cls, generated_tags):
```

Comment:
This should be either an `@staticmethod` of a separate function.

Reply:
This method should be in the _helpers IMO.

Reply:
At first I put it in _deserialize.py, while  _deserialize is importing stuff from  _model.py so model.py cannot import stuff from _deserialize.
_parse_tags is used by line 545, so it makes sense to leave _parse_tags as a staticmethod.